apps/condo/domains/common/constants/webhooks.js

```diff
@@ -0,0 +1,10 @@
+const WEBHOOK_EVENT_PAYMENT_STATUS_UPDATED = 'payment.status.updated'
```

Comment:
It's just one event. It looks like you dont need it at all.

Reply:
This is an application-level webhook event. For now, this is the first event in this app.